apps/emqx_auth_jwt/src/emqx_auth_jwt.appup.src

```diff
@@ -1,16 +1,9 @@
 %% -*- mode: erlang -*-
 %% Unless you know what you are doing, DO NOT edit manually!!
 {VSN,
- [
-    {<<"4\\.4\\.[0-1]">>, [
-      {restart_application,emqx_auth_jwt}
-    ]},
-    {<<".*">>, []}
- ],
- [
-    {<<"4\\.4\\.[0-1]">>, [
-      {restart_application,emqx_auth_jwt}
-    ]},
-    {<<".*">>, []}
- ]
-}.
+  [{"4.4.2",[{load_module,emqx_auth_jwt,brutal_purge,soft_purge,[]}]},
+   {<<"4\\.4\\.[0-1]">>,[{restart_application,emqx_auth_jwt}]},
```

Comment:
Nit:

```suggestion
   {<<"4\\.4\\.[0-1]$">>,[{restart_application,emqx_auth_jwt}]},
```

Reply:
Yes they are

Reply:
I thought they might match only the prefix.  Ignore me then.  :speak_no_evil: